modules/apps/object/object-test/src/testFunctional/macros/ObjectModelBuilder.macro

```diff
@@ -1,5 +1,51 @@
 definition {
 
+	@summary = "Default summary"
+	macro addObjectRelationshipViaUIInModelBuilder(invalidEntry = null, parameterOption = null, name = null, label = null, type = null, object = null,) {
+		Click(
+			key_kebabOption = "Add Relationship",
+			locator1 = "ObjectAdmin#KEBAB_MENU_OPTION");
+
+		SelectFrame(locator1 = "ObjectCustomViews#VIEW_MODAL_CONTENT");
+
+		Type(
+			locator1 = "ObjectAdmin#LABEL_OBJECT_RELATIONSHIP",
+			value1 = ${label});
+
+		if (isSet(type)) {
+			Click(locator1 = "ObjectAdmin#RELATIONSHIPS_TYPE_FORM");
+
+			Click(
+				key_type = ${type},
+				locator1 = "ObjectAdmin#SELECT_RELATIONSHIPS_TYPE_OPTION");
+		}
+
+		if (isSet(object)) {
+			Click(locator1 = "ObjectCustomViews#SELECT_AUTO_COMPLETE");
+
+			Click(
+				key_columnOption = ${object},
+				locator1 = "ObjectCustomViews#SELECT_AUTO_COMPLETE_OPTION");
+		}
+
+		if (isSet(name)) {
+			Type(
+				locator1 = "ObjectAdmin#NAME_OBJECT_RELATIONSHIP",
+				value1 = ${name});
```

Comment:
@thaiscabrals relationshipName and relationshipType might be best too so we can easily tell what we are updating just by reading the test. Btw, it might make it more readable too if we edit Name before Type so it's consistent with the order in the modal. Lmk if you have any questions!